logs_http.module

```diff
@@ -136,7 +136,8 @@ function logs_http_register_event(array $log_entry) {
     'uid' => $log_entry['uid'],
     'link' => strip_tags($log_entry['link']),
     'message' => empty($log_entry['variables']) ? $log_entry['message'] : strtr($log_entry['message'], $log_entry['variables']),
-    'severity' => $log_entry['severity'],
+    'severity' => watchdog_severity_levels()[$log_entry['severity']],
```

Comment:
Ah, ok! Didn't know this, but it's a fast change we can do! I'll note it down in my PHP knowledge!

I'm not a super expert in PHP, but I have some expertise in multiple languages, so I still miss this things :)